index.html

```diff
@@ -50,5 +50,173 @@ <h1>I’m Mohammed Glad to see you!</h1>
         </ul>
       </div>
     </section>
+
+    <section class="work-section">
+      <div class="card1 card-work">
+        <img src="/img/Snapshoot-Portfolio.svg" alt="" />
+        <h1>Tonic</h1>
+        <div class="frame2">
+          <p>CANOPY</p>
+          <ul>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+        </div>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="programing-lamguages">
+          <li>html</li>
+          <li>css</li>
+          <li>javaScript</li>
+        </ul>
+        <button type="submit">See Project</button>
+      </div>
```

Comment:
- [ ] Please do your best to follow this design on Figma and make your button look the same as on the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)
- [ ] Your design
![yoursdesign](https://user-images.githubusercontent.com/55596970/153619988-27b4d666-14dd-4d76-a493-a6de0c109010.png)

- [ ] Figma design 

![requireDesign](https://user-images.githubusercontent.com/55596970/153620107-5c6c24d5-61d1-4c31-8d71-19a287a4e458.png)

Reply:
![figma](https://user-images.githubusercontent.com/97116942/153621898-a5ef3d17-d999-47b6-af27-e7804d8f3e13.PNG)

my button looks the same as on the Figma template